provisioning/transport/http/src/ProvisioningErrorDetailsHttp.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
Here and other places: please ensure that you have added the proper file headers.

Reply:
Sure thing